src/app/public/modules/sky-tree-view/sky-tree-view-context-menu.component.ts

```diff
@@ -0,0 +1,15 @@
+import {
+  ChangeDetectionStrategy,
+  Component,
+  ViewEncapsulation
+} from '@angular/core';
+
+@Component({
+  selector: 'sky-tree-view-context-menu',
+  templateUrl: './sky-tree-view-context-menu.component.html',
+  encapsulation: ViewEncapsulation.None,
```

Comment:
You decided to go with a global style sheet, instead of overwriting the styles from the `sky-tree-view` component?

Reply:
Whoops, that was copy-pasta. I completely looked passed this, thanks for catching that.
I've updated this to use a component style sheet.